composer.json

```diff
@@ -39,6 +39,7 @@
         },
         "files": [
             "src/Core/functions.php",
+            "src/Core/deprecated.php",
```

Comment:
How is this going to work for each of the subsplits? Won't those repos have issues now?